cg/cli/workflow/commands.py

```diff
@@ -112,9 +119,33 @@ def store_available(context: click.Context, dry_run: bool) -> None:
         raise click.Abort
 
 
+@click.command("clean-rsync-dirs")
+@OPTION_YES
+@OPTION_DRY
+@click.pass_obj
```

Comment:
Why not keep the "before_str" argument so that you modulate the removal date easily? It will also behave as the other clean commands

Reply:
Yes I could add it again. I did remove it here because I was not actually using it in the code, but if we want to change it I agree it would be easier to change it in servers than in this script.

Reply:
Great! I think it would be more flexible and standardised

Reply:
fixed